ipld/src/json.rs

```diff
@@ -1,6 +1,18 @@
 // Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
+//! Differences between serializers
+//!
+//! The serializer created here uses `multihash` and `libipld-json` uses plain `base64`.
+//! That means one has an extra `m` in front of all the encoded byte values, using our serializer.
+//!
+//! For example:
+//!
+//! |`forest_ipld`|`libipld-json`|
+//! | ----------- | ------------ |
+//! |```{ "/": { "bytes": "mVGhlIHF1aQ" } }```|```{ "/": { "bytes": "VGhlIHF1aQ" } }```|
+//!
+//! Since `Lotus` is also using `multihash-base64` and we're trying to be compatible, we cannot switch to `libipld-json`. It may be worthwhile to reconsider whether we want to stay compatible with Lotus in the future.
```

Comment:
Could you wrap this line so it's no longer than 100 characters?

I tried enforcing this automatically with `cargo fmt` but I wouldn't find an option for it. Strange.